tests/e2e/support/page/files/version.ts

```diff
@@ -28,4 +39,33 @@ export class VersionPage {
     const elements = this.fileVersionSidebarXPath
     return await elements.count()
   }
+
+  async restoreOlderVersion({ files, folder }: { files: File[]; folder?: string }) {
+    const { page } = this.actor
+    const { dir: resourceDir, base: resourceBase } = path.parse(folder)
```

Comment:
I'm a little confused as to why this is necessary?

Reply:
Sorry, I forgot to remove that line.